python/lsst/cp/pipe/utils.py

```diff
@@ -500,7 +500,14 @@ def arrangeFlatsByExpFlux(exposureList, exposureIdList, fluxKeyword):
     assert len(exposureList) == len(exposureIdList), "Different lengths for exp. list and exp. ID lists"
     for expRef, expId in zip(exposureList, exposureIdList):
         # Get flux from header, assuming it is in the metadata.
-        expFlux = expRef.get().getMetadata()[fluxKeyword]
+        try:
+            expFlux = expRef.get().getMetadata()[fluxKeyword]
```

Comment:
Is this a keyword written by some other task or something that comes from a raw header? Should we issue a log message if you think this should be there?

Reply:
This is logged as missing afterwords.  I didn't want to add a log in here just to get things unblocked.

But it turns out the source of this problem is that one of the data taking controllers was giving corrupt header information and what got into the repo depended on the order of ingestion of the different detectors.  Not great!